Cargo.toml

```diff
@@ -0,0 +1,19 @@
+[package]
+name = "deprecated-api-versions"
+version = "0.1.0"
```

Comment:
I wonder if we could use `0.1.0+k8sv1.25.0`, to denote up until what deprecations in which Kubernetes versions the policy knows about.
(using `+` in semver is reserved for build info, which is not super fitting, but better than `-` which is for prereleases).

Reply:
Oh... this is a nice touch. I like it!

Reply:
I've done that with e554c3e3965c05b987a3a61df0557b0aed2cef33

Can you please take another look?

Reply:
Looks nice! approving, just a little typo.